src/inmanta/protocol/openapi/model.py

```diff
@@ -65,11 +65,14 @@ class SchemaBase(BaseModel):
     readOnly: Optional[bool] = None
     example: Optional[Any] = None
     deprecated: Optional[bool] = None
+    anyOf: Optional[List[Any]] = None
```

Comment:
```suggestion
    anyOf: Optional[List["Schema"]] = None
```
Not entirely sure about this but if I understand correctly this is it's purpose. At least it doesn't give any mypy errors.